src/components/Chart/util.test.ts

```diff
@@ -0,0 +1,146 @@
+import { getDomainMaxValue, isOutlier, getOutlierStep, getOutlierSeriesName, getOutlierStepFromDomain } from './util'
+import { AxisDomain, ValueRange, DateRange, DataPoint } from './model'
+
+describe('getDomainMaxValue', () => {
+  it.each([null, undefined])('should return null when domain is null or undefined', (domain) => {
+    const result = getDomainMaxValue((domain as unknown) as AxisDomain)
+
+    expect(result).toBeNull()
+  })
+
+  it('should return the domain end value when domain is an valid object', () => {
+    const domain: AxisDomain = { init: 10, end: 10, step: 10 }
+
+    const result = getDomainMaxValue(domain)
+
+    expect(result).toBe(10)
```

Comment:
```suggestion
    const domain: AxisDomain = { init: 10, end: 15, step: 10 }

    const result = getDomainMaxValue(domain)

    expect(result).toBe(15)
```

Reply:
I think it's better to be different from the others values

Reply:
Done!